### ISSUE TITLE
Samplerate issue on core i3-5 series / segfault

### ISSUE THREAD

**Issue by majjam** [nonlinear-labs-dev/C15#2412]
Sometimes I have audio performance issues 
Have in mind that I do have a very early board settup with i3-5

Dmesg shows this :

[   13.664218] snd_hda_intel 0000:00:1b.0: IRQ timing workaround is activated for card #2. Suggest a bigger bdl_pos_adj.
[  710.972715] perf: interrupt took too long (2507 > 2500), lowering kernel.perf_event_max_sample_rate to 79500
[ 1158.676847] perf: interrupt took too long (3140 > 3133), lowering kernel.perf_event_max_sample_rate to 63600
                                                                                                
I saw a CPU dependant buffer setting (latency) in the audioengine-service and thought i3-5 was supported. Am I wrong here ?

**Comment by majjam** [comment]
Here is another one from today:
this leads to a restart of the audioengine.

[  741.301090] perf: interrupt took too long (2502 > 2500), lowering kernel.perf_event_max_sample_rate to 79800
[  930.057592] WebSockOut[876]: segfault at 45 ip 0000000000000045 sp 00007f5146ffcc38 error 14 in audio-engine[55efc8670000+22000]
[  930.057663] audio-engine[879]: segfault at 7f511c0049b0 ip 00007f511c0049b0 sp 00007f51451f0a58 error 15

**Comment by majjam** [comment]
Sorry this issue was a false alarm !
I misinterpreted the kernel message. i learned that performance monitor is watched by the kernel and it self adjusts the per mon sample rate. ithought it had to do with the audio sample rate.

I would ignore the segfaults here as i saw it only once on my own build where libplayground could be not 100% matching to plagyound (not sure here)
If i see any more segfaults i will open a new issue.
thanks 
Max
